File: sdkwork-ai-sdk-kotlin/src/main/kotlin/com/sdkwork/ai/DocumentResponse.kt
package com.sdkwork.ai

data class DocumentResponse(
    val title: String? = null,
    val type: String? = null,
    val status: String? = null,
    val metadata: Map<String, Any>? = null,
    val documentId: String? = null,
    val knowledgeBaseId: String? = null,
    val chunkCount: Int? = null,
    val createdAt: Int? = null,
    val updatedAt: Int? = null
)

File: sdkwork-ai-sdk-python/sdkwork_ai_sdk/models/document_response.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class DocumentResponse:
    """文档响应"""
    title: str = None
    type: str = None
    status: str = None
    metadata: Dict[str, Any] = None
    document_id: str = None
    knowledge_base_id: str = None
    chunk_count: int = None
    created_at: int = None
    updated_at: int = None
